lib/Echidna/Config.hs

```diff
@@ -59,14 +60,16 @@ instance FromJSON EConfig where
                           <*> v .:? "seqLen"      .!= 100
                           <*> v .:? "shrinkLimit" .!= 5000
                           <*> pure Nothing
+                          <*> v .:? "seed"
         names = const $ const mempty :: Names
-        ppc = cc <&> \c x -> runReader (ppCampaign x) (c, names)
-        style :: Y.Parser (Campaign -> String)
+        ppc = cc <&> \c x _ -> runReader (ppCampaign x) (c, names)
+        style :: Y.Parser (Campaign -> Maybe Int -> String)
```

Comment:
IMO, we should always know what the seed is here. I'm not sure why we need to handle the "Maybe" case

Reply:
fixed